server/webapp/WEB-INF/rails.new/lib/message_verifier.rb

```diff
@@ -0,0 +1,27 @@
+##########################################################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+class MessageVerifier
+  def self.verify(message, signature, public_key_path)
```

Comment:
Can we add another helper method that accepts a public key string, to make it easier for testing? I see you're having to stub file operations.